spec/features/users/profile_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+
+# As a registered user
```

Comment:
Do we want to remove these in the future?

Reply:
Yeah, I just forgot to remove these when I made the PR. Thanks for reminding me!